airflow/providers/amazon/aws/operators/batch.py

```diff
@@ -190,11 +197,31 @@ def execute(self, context: Context):
         """
         self.submit_job(context)
 
+        if self.deferrable:
+            self.defer(
+                timeout=self.execution_timeout,
+                trigger=BatchOperatorTrigger(
+                    job_id=self.job_id,
+                    max_retries=self.max_retries or 10,
```

Comment:
nit: I would make this consistent with how `poll_interval` is done (i.e. assign a default value in `init`. But that's just my opinion.

Reply:
I understand your concern.
currently, max_reries is of type optional init and in BatchClientHook we do` self.max_retries = max_retries or self.MAX_RETRIES` I feel we should change the type to int in init something like `def __init__(self, max_retries: int = MAX_RETRIES, ...)` but is it ok if we take it in a separate PR?

Reply:
Sounds good!